plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHiveIntegrationSmokeTest.java

```diff
@@ -7319,6 +7319,25 @@ public void testSortedWritingTempStaging()
         assertUpdate("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testTrinoSortBy()
+    {
+        String tableName = "test_trino_sort";
+        @Language("SQL") String createTableSql = format("" +
+                        "CREATE TABLE %s " +
+                        "WITH (" +
+                        "   trino_sort_by = ARRAY['suppkey']" +
+                        ") AS " +
+                        "SELECT * FROM tpch.tiny.lineitem",
```

Comment:
please use small table (e.g nation) and validate order (entire nation table should be read in one batch). Even better would be to run `select * from test_trino_sort order by XXX` and validate that there is no sort in plan (but I don't think it's supported now). Alternatively, some other operator can be used that takes sorting order into account.

There are plan validatign tests in Hive integration smoke tests (`io.trino.plugin.hive.TestHiveIntegrationSmokeTest#assertRemoteExchangesCount`)

Reply:
Changed test to use nation, can't do the planner related things since we're not going to use sort_by for planner optimisations based on https://github.com/trinodb/trino/pull/6626#discussion_r559512853